src/components/LabeledRadioButton/examples/interactive.jsx

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash';
+import React from 'react';
+
+import LabeledRadioButton from '../LabeledRadioButton';
+
+const style = {
+	marginRight: '5px'
+};
+
+export default React.createClass({
+	getInitialState() {
+		return {
+			flavor: 'vanilla'
+		};
+	},
+
+	handleSelectedChocolate() {
+		this.setState(_.assign({}, this.state, {
```

Comment:
the `_.assign` is redundant since `this.setState` already does an implicit assign